scripts/scheduler.py

```diff
@@ -374,6 +375,7 @@ def queue_new_jobs(available_slots, env, config, studies, wes_token, exclude_stu
           try:
               run_info = get_run_state(graphql_url, latest_run_id, wes_token)
               resume = run_info['sessionId'] if run_info.get('sessionId') else False
+              work_dir = run_info['engineParameters']['workDir']
```

Comment:
we could extract `nfs` path from the `workDir` path, then we reassign `nfs` variable here to match want was use for the job in the last run.

Reply:
@junjun-zhang but `workflow_version` is also part of the path for `work_dir` and `lauch_dir`.

Reply:
ok, I see